src/components/Spinner/index.jsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import PropTypes from 'proptypes';
+import './styles.scss';
+
+const Spinner = ({ isAbsolute, size }) => (
+  <div className={`spinner ${isAbsolute ? 'large' : size}`} />
+);
+
```

Comment:
Looking at the code is seems that size is the determining factor for the class to be added so I think that size could be given a default value of 'large' and should be optional if it has a default value e.g
```
<div className={`spinner ${size}`} />
```
```
Spinner.propTypes = {
  size: PropTypes.string,
};
Spinner.defaultProps{
    size: 'large'
}
```
or
```
const Spinner = ({ size='large }) => (
  <div className={`spinner ${size}`} />
);
```